### PR TITLE
Completed Ip_route parser

### PR DESCRIPTION
**Pull request by Julian5555**
Completed the ip_route parser. Let me know if any formatting or changes need to be made.

### CODE REVIEW

File: jc/parsers/ip_route.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+"""jc - JSON Convert `ip route` command output parser
+
+Usage (cli):
+
+    $ ip route | jc --ip_route
+
+or
+
+    $ jc ip_route
```
Comment (kellyjonbrazil) [inline jc/parsers/ip_route.py:9]: The `jc` argument will be `ip-route` with a dash here.

**Review by kellyjonbrazil — CHANGES_REQUESTED** [review]
Nice work - thanks for the contribution!

**Review by Julian5555 — COMMENTED** [review]
I reformatted the requested changes and changed the metric key into an int for non raw data while leaving r data as a dictionary of strings as per you're request. Also used if's instead of the previous match cases so that the parser should now be backwards compatible. If I missed anything else, please let me know. Thanks for the review.

**Review by kellyjonbrazil — APPROVED** [review]

**Comment by kellyjonbrazil** [comment]
Thanks for the updates! I'll merge and do a little cleanup.

**Comment by Julian5555** [comment]
Looks great, I appreciate the formatting help and you taking the time to review this.

**Pull request merged**